### PR TITLE
Update text for Set-CsOnlineEnhancedEmergencyServiceDisclaimer

### PR DESCRIPTION
**Pull request by JensTrier**
Updated the description and synopsis to make it clearer what this cmdlet does

### CODE REVIEW

File: skype/skype-ps/skype/Set-CsOnlineEnhancedEmergencyServiceDisclaimer.md [hunk 1]
```diff
@@ -13,8 +13,16 @@ ms.reviewer:
...
+When using Microsoft Teams PSTN Calling Services you need to record your organization's acceptance of the enhanced emergency service terms and conditions. This is done per
+country and it needs to be done before you can provide PSTN calling services to Microsoft Teams users in the country.
+
+You can record your organization's acceptance using the Set-CsOnlineEnhancedEmergencyServiceDisclaimer cmdlet at any time. If you haven't accepted it for a given country
+you will be prompted to do so by warning information in the Teams PS Module, when you try to assign a phone number to Microsoft Teams user, or in the Teams admin center,
+when you create an emergency address in a country.
+
+Any tenant administrator can accept the terms and conditions and it only needs to done once per country.
+
+As the output the cmdlet will show the emergency service disclaimer and that it has been accepted. You can use Get-CsOnlineEnhancedEmergencyServiceDisclaimer to see the status of the emergency service disclaimer.
```
Comment (dariomws) [inline skype/skype-ps/skype/Set-CsOnlineEnhancedEmergencyServiceDisclaimer.md:25]:
```suggestion
When using Microsoft Teams PSTN Calling Services you need to record your organization's acceptance of the enhanced emergency service terms and conditions. This is done per
country and it needs to be done before you can provide PSTN calling services to Microsoft Teams users in the country.

You can record your organization's acceptance using the Set-CsOnlineEnhancedEmergencyServiceDisclaimer cmdlet at any time. If you haven't accepted it for a given country
you will be prompted to do so by warning information in the Teams PS Module, when you try to assign a phone number to a Microsoft Teams user, or in the Teams admin center,
when you create an emergency address in a country.

Any tenant administrator can accept the terms and conditions and it only needs to be done once per country.

As the output the cmdlet will show the emergency service disclaimer and that it has been accepted. You can use Get-CsOnlineEnhancedEmergencyServiceDisclaimer to see the status of the emergency service disclaimer.
```

File: skype/skype-ps/skype/Set-CsOnlineEnhancedEmergencyServiceDisclaimer.md [hunk 2]
```diff
@@ -23,9 +31,11 @@ Set-CsOnlineEnhancedEmergencyServiceDisclaimer -CountryOrRegion <String> [-Versi
  [-Tenant <Guid>] [-DomainController <Fqdn>] [-Force] [-WhatIf] [-Confirm] [<CommonParameters>]
 ```
 
-
 ## DESCRIPTION
-You must run this cmdlet prior to assigning phone numbers and locations to voice enabled users in the United States.
+You must run this cmdlet prior to assigning Microsoft Calling Plan phone numbers and locations to voice enabled users or accept the similar disclaimer in the Teams admin center.
+
+Microsoft Calling Plan phone numbers are available in several countries, see this link https://docs.microsoft.com/MicrosoftTeams/country-and-region-availability-for-audio-conferencing-and-calling-plans/country-and-region-availability-for-audio-conferencing-and-calling-plans
```
Comment (get-itips) [inline skype/skype-ps/skype/Set-CsOnlineEnhancedEmergencyServiceDisclaimer.md:37]:
```suggestion
Microsoft Calling Plan phone numbers are available in several countries, see [Country and region availability for Audio Conferencing and Calling Plans](https://docs.microsoft.com/MicrosoftTeams/country-and-region-availability-for-audio-conferencing-and-calling-plans/country-and-region-availability-for-audio-conferencing-and-calling-plans)
```

File: skype/skype-ps/skype/Set-CsOnlineEnhancedEmergencyServiceDisclaimer.md [hunk 3]
```diff
@@ -41,14 +51,13 @@ version of the enhanced emergency service terms and conditions.
...
-Specifies the region or country whose terms and conditions you wish to accept.
-The United States is currently the only country supported, but it must be specified as "US".
+Specifies the region or country whose terms and conditions you wish to accept. You need to use the ISO 31661-1 alpha-2 2 letter code for the country. For example for the United States it must be specified as "US" and for Denmark it must be specified as "DK".
 
 ```yaml
 Type: String
 Parameter Sets: (All)
 Aliases: 
-Applicable: Skype for Business Online
+Applicable: Skype for Business Online and Microsoft Teams
```
Comment (get-itips) [inline skype/skype-ps/skype/Set-CsOnlineEnhancedEmergencyServiceDisclaimer.md:60]:
```suggestion
Applicable: Skype for Business Online, Microsoft Teams
```

Comment (get-itips) [reply skype/skype-ps/skype/Set-CsOnlineEnhancedEmergencyServiceDisclaimer.md:60]: This field needs commas

File: skype/skype-ps/skype/Set-CsOnlineEnhancedEmergencyServiceDisclaimer.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 ---
 external help file: Microsoft.Rtc.Management.Hosted.dll-help.xml 
 online version: https://docs.microsoft.com/powershell/module/skype/set-csonlineenhancedemergencyservicedisclaimer
-applicable: Skype for Business Online
+applicable: Skype for Business Online and Microsoft Teams
```
Comment (get-itips) [inline skype/skype-ps/skype/Set-CsOnlineEnhancedEmergencyServiceDisclaimer.md:4]:
```suggestion
applicable: Skype for Business Online, Microsoft Teams
```

**Review by dariomws — COMMENTED** [review]
Some minor suggestions.

Not sure about add Teams to the applicable section. All the other cmdlets only include Skype for Business Online.

**Comment by JensTrier** [comment]
> Some minor suggestions.
> 
> Not sure about add Teams to the applicable section. All the other cmdlets only include Skype for Business Online.

I can remove Microsoft Teams from Applicable section, just let me know

**Review by get-itips — COMMENTED** [review]
Please correct the other `Applicable` occurrences, thanks

**Review by tseward — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Comment by dariomws** [comment]
> > Some minor suggestions.
> > Not sure about add Teams to the applicable section. All the other cmdlets only include Skype for Business Online.
> 
> I can remove Microsoft Teams from Applicable section, just let me know

Let's add it, there are a few more cmdlets with the same information and I think it's a good idea.

**Comment by JensTrier** [comment]
> > > Some minor suggestions.
> > > Not sure about add Teams to the applicable section. All the other cmdlets only include Skype for Business Online.
> > 
> > 
> > I can remove Microsoft Teams from Applicable section, just let me know
> 
> Let's add it, there are a few more cmdlets with the same information and I think it's a good idea.

I added it.

**Comment by JensTrier** [comment]
@get-itips  do you need more changes?

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**